crates/agglayer-node/src/rpc/mod.rs

```diff
@@ -229,7 +229,7 @@ where
             return Err(Error::rollup_not_registered(tx.tx.rollup_id));
         }
 
-        self.kernel.verify_signature(&tx).await.map_err(|e| {
+        self.kernel.verify_tx_signer(&tx).await.map_err(|e| {
```

Comment:
The naming changes should probably be reverted. Verifying tx signer implies signature validity.